Detectors/TRD/workflow/src/TRDDigitizerSpec.cxx

```diff
@@ -146,8 +146,9 @@ class TRDDPLDigitizerTask : public o2::base::BaseDPLDigitizer
     if (mctruth) {
       labelsAccum.mergeAtBack(labels);
     }
+    LOGF(INFO, "List of TRD chambers for which at least once an unusual drift velocity was found: %s", mDigitizer.dumpFlaggedChambers());
```

Comment:
I think that `"List of TRD chambers with at least one drift velocity out-of-range: %s"` is more consistent with what the real error is.

Reply:
Thanks, changed